Documentation/CodingGuidelines/CglPhp/PhpFileFormatting/PhpSyntaxFormatting.rst

```diff
@@ -64,7 +64,6 @@ characters.
 Examples::
```

Comment:
either find another Eyample or change "Examples" to singular

Reply:
ok. Will do.